src/bootstrap/tool.rs

```diff
@@ -254,6 +254,13 @@ macro_rules! bootstrap_tool {
             }
 
             fn run(self, builder: &Builder<'_>) -> PathBuf {
+                if $tool_name == "compiletest" {
```

Comment:
Can we limit builds to only those cases where we actually need the run-make support? i.e., run-make executions of compiletest?

Reply:
I've attempted to have `rmake_support` only be built when `run-make` is about to be executed in 5279d4c. I'm not 100% sure if what I wrote is correct, but at the very least it works on my machine™...